manifests/0000_31_cluster-baremetal-operator_05_prometheus_rbac.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
same with this

Reply:
Done. Do we still need the above change in controllers/provisioning_controller.go then?

Reply:
no, i think you can remove it.